doc/RelNotes.md

```diff
@@ -3,6 +3,27 @@ Release Notes {#ReleaseNotes}
 
 # New in this release {#New}
 
+DisplayCluster 0.6 introduces a new focus mode for windows.
+
+## New Features {#NewFeatures}
+
+* Double tapping a window bring it in focus mode, replacing the old confusing
+  switch to and from interaction mode with a tap and hold.
+* Animated transitions to and form the focus mode.
```

Comment:
Sorry, I hadn't pushed the code review commit. I was waiting to include the Deflect update.